### ISSUE TITLE
Add Changelog Entry & Tags for v2.0.1 & 2.0.2

### ISSUE THREAD

**Issue by westonganger** [caxlsx/caxlsx#31]
Both 2.0.1 and 2.0.2 are missing both Tags and Changelog entries. Please add these.

v2.0.1 - https://github.com/randym/axlsx/commit/46a17e4b3fe0531ac6b2e48f818a982a86f538e4

v2.0.2 - specifically I would like a changelog entry to specify what this release includes, im not even sure what commit this release is based off of

- https://rubygems.org/gems/caxlsx/versions/2.0.2
- https://rubygems.org/gems/axlsx/versions/2.0.1 - has a different rubyzip dependency than 2.0.2

**Comment by noniq** [comment]
I tagged 97f7735 as [v2.0.1](https://github.com/caxlsx/caxlsx/tree/v2.0.1). This is the commit the [2.0.2 branch](https://github.com/caxlsx/caxlsx/tree/2.0.2) is based off.

I tagged 86a40d52 as [v2.0.2](https://github.com/caxlsx/caxlsx/tree/v2.0.2). There are some later commits in the 2.0.2 branch, but they are not touching any files included in the released gem. 

The changelog entry for 2.0.1 is present in the master branch, and I just added an entry for 2.0.2 too: 45a091e

Unfortunately these entries (as well as those for 1.3.5 and 1.3.6) were missing from the changelog in the 2.0.2 branch, which means that they are also not part of the released gem versions 2.0.1 and 2.0.2.

I added those entries in the 2.0.2 branch just now (b2fa88a9). If we want to be pedantic we could even release 2.0.3 at this point, to have a 2.x version out there which includes the full changelog. Does anyone think this is necessary?
